src/sentry/runner/commands/configoptions.py

```diff
@@ -136,13 +126,15 @@ def configoptions(ctx, dry_run: bool, file: Optional[str], hide_drift: bool) ->
     ctx.obj["options_to_update"] = options_to_update
 
     drifted_options = set()
+    presenter_delegator = PresenterDelegator()
+    ctx.obj["presenter_delegator"] = presenter_delegator
+
     for key, value in options_to_update.items():
         not_writable_reason = options.can_update(key, value, options.UpdateChannel.AUTOMATOR)
```

Comment:
You'll have to catch exceptions here as well, as, if the option does not exist this line will fail before you reach line 182.
See https://github.com/getsentry/sentry/pull/53814/files for where errors conditions can arise.